src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocmethod/Example1.java

```diff
@@ -0,0 +1,41 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="JavadocMethod"/>
+  </module>
+</module>
+*/
+package com.puppycrawl.tools.checkstyle.checks.javadoc.javadocmethod;
+
+// xdoc section -- start
+public class Example1 {
+
+  /**
+   *
+   */
+  Example1(int x) { // violation, param tag missing for x
```

Comment:
Please do compact form of ctor 
`Example1(int x) {}`

Please use violation message in trailing comments, with single quotes.

Please make javadoc to be compact `/** */`
Please put violation comment under ctor, to be horizontally short.

Reply:
Done